fhir-persistence-schema/src/test/java/com/ibm/fhir/schema/prior/FhirSchemaGenerator455.java

```diff
@@ -0,0 +1,949 @@
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.RESOURCE_TYPES;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.RESOURCE_TYPE_ID;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.STR_VALUE;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.STR_VALUES;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.STR_VALUE_LCASE;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANTS;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_HASH;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_KEYS;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_KEY_ID;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_NAME;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_SALT;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_SEQUENCE;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_STATUS;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.TOKEN_VALUE;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.TOKEN_VALUES;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.ibm.fhir.database.utils.api.IDatabaseStatement;
+import com.ibm.fhir.database.utils.common.AddColumn;
+import com.ibm.fhir.database.utils.common.CreateIndexStatement;
+import com.ibm.fhir.database.utils.common.DropColumn;
+import com.ibm.fhir.database.utils.common.DropIndex;
+import com.ibm.fhir.database.utils.model.AlterSequenceStartWith;
+import com.ibm.fhir.database.utils.model.BaseObject;
+import com.ibm.fhir.database.utils.model.ColumnBase;
+import com.ibm.fhir.database.utils.model.ColumnDefBuilder;
+import com.ibm.fhir.database.utils.model.FunctionDef;
+import com.ibm.fhir.database.utils.model.Generated;
+import com.ibm.fhir.database.utils.model.GroupPrivilege;
+import com.ibm.fhir.database.utils.model.IDatabaseObject;
+import com.ibm.fhir.database.utils.model.NopObject;
+import com.ibm.fhir.database.utils.model.ObjectGroup;
+import com.ibm.fhir.database.utils.model.OrderedColumnDef;
+import com.ibm.fhir.database.utils.model.PhysicalDataModel;
+import com.ibm.fhir.database.utils.model.Privilege;
+import com.ibm.fhir.database.utils.model.ProcedureDef;
+import com.ibm.fhir.database.utils.model.Sequence;
+import com.ibm.fhir.database.utils.model.SessionVariableDef;
+import com.ibm.fhir.database.utils.model.Table;
+import com.ibm.fhir.database.utils.model.Tablespace;
+import com.ibm.fhir.model.type.code.FHIRResourceType;
+import com.ibm.fhir.schema.control.FhirSchemaConstants;
+import com.ibm.fhir.schema.control.FhirSchemaVersion;
+import com.ibm.fhir.schema.control.SchemaGeneratorUtil;
+
+/**
+ * Encapsulates the generation of the FHIR schema artifacts
+ */
+public class FhirSchemaGenerator455 {
```

Comment:
copyright

Reply:
Although this is a new class by name, it's a complete copy of FHIRSchemaGenerator from a prior release, so I'm keeping the 2019 as its creation date.